alpaca/data/live/news.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional, Dict
+
+from alpaca.common.enums import BaseURL
+from alpaca.common.websocket import BaseStream
+from alpaca.data.enums import DataFeed
+
+
+class NewsDataStream(BaseStream):
+    """
+    A WebSocket client for streaming live news feed from alpaca data api.
+
+    See BaseStream for more information on implementation and the methods available.
+    """
+
+    def __init__(
+        self,
+        api_key: str,
+        secret_key: str,
+        raw_data: bool = False,
+        feed: DataFeed = DataFeed.NEWS,
```

Comment:
DataFeed is used for equity. Therefore, it may be good not to use DataFeed.
Can we hardcode the url path (/news) and remove feed parameter here?